dbt/dbt_project.yml

```diff
@@ -1,6 +1,5 @@
 name: 'athena'
 version: '0.1.0'
-config-version: 2
```

Comment:
This option and the `version: 2` key are [optional as of dbt 1.5](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.5#quick-hits). Exclude 4c15ff7 to see only schema structure changes.